### ISSUE TITLE
Marriage Certificate issues (SK)

### ISSUE THREAD

**Issue by michal-o** [SEMICeu/SDG-sandbox#41]
We don't specifically record "capacity to marry". What is meant by this attribute?

**Comment by sethvanhooland** [comment]
Thank you for your question Michal. This refers to the legal conditions under which someone is allowed to marry, such as being at least 18 years old.

**Comment by pfragkou** [comment]
To complement on this, in Greece in order a person below 18 years old to get married, parent's written consent is needed.

**Comment by ibodor** [comment]
**Ingrid (FIN):** I believe there is some contradiction here. We are talking about marriage certificate and there is a hidden assumption here that the requestor's marital status is married at the time of the request. Therefore information about capacity to marry is unnecessary/futile. In Finland it is possible to order the certificate of the current marriage with history data (= previous marriages are also shown if requested) - should it be clarified what marriage certificate we are talking about? the current marriage of the requestor? the last marriage (may be currently single/divorced)? Or is it actually the marital status that we are after ? (in this case capacity to marry could be relevant)

**Comment by cbahim** [comment]
Thanks for your comment Ingrid! To answer your question, we are talking about the current marriage of the requestor. Information about the last marriage are captured by `maritalStatusBeforeMarriage`. What would be your opinion with regards to the `capacityToMarry` attribute then?

**Comment by cbahim** [comment]
As mentioned in issue #38 , ES does not record information on `citizenship`, `capacityToMarry` and `maritalStatusBeforeMarriage`

> we don't have in our marriage records information on the "citizenship", "capacityToMarry" or "maritalStatusBeforeMarriage" of persons

**Comment by ibodor** [comment]
> Thanks for your comment Ingrid! To answer your question, we are talking about the current marriage of the requestor. Information about the last marriage are captured by `maritalStatusBeforeMarriage`. What would be your opinion with regards to the `capacityToMarry` attribute then?

**Ingrid (FIN):** ok, so if the requirement is "getCurrentMarriageOfCustomer" or something along these lines, then we should  get back either the marriage certificate dataset we are discussing here (=marital status at the time of request is married) or we don't get back anything (=not married at the time of the request). In the first case(= we get data back) capacityToMarry is futile, in the latter one we would not have data at all, so the capacityToMarry value could be deducted as YES=could get married. I think we need to clarify the use case here, what i just described would be the use case "get the data of the current marriage of person X" and not get this person's marital status or marriage history or marital state history. 
I still somehow suspect, that the actual info we are after here is the current marital status, capacityToMarry could be deducted from this value. And marital status itself could be deducted based on "getCurrentMarriageOfCustomer" dataset returned /not returned. I am not saying this is the most elegant way to do it, but logically should be correct.
In the Finnish certificate previous marriages can be shown and this data does not fit the maritalStatusBeforeMarriage attribute you suggested, some calculation and mapping would be needed here. Not impossible, just some extra work - in case it is justified.

So to sum up: what info do we actually want here, what are we going to use it for? (e.g. we want to know if person A is legally allowed to get married at the time of the request, we want a proof that person A is indeed married to person B, etc)

**Comment by cbahim** [comment]
`capacityToMarry` cannot be deducted from whether or not you get information about the current marriage. As mentioned [above](https://github.com/SEMICeu/SDG-sandbox/issues/41#issuecomment-664150554) `capacityToMarry` is about the legal conditions under which someone is allowed to marry (e.g. legal age, no known impediment). Further information can be found in the [Public document forms](https://beta.e-justice.europa.eu/35981/EN/public_documents_forms)

> So to sum up: what info do we actually want here, what are we going to use it for? (e.g. we want to know if person A is legally allowed to get married at the time of the request, we want a proof that person A is indeed married to person B, etc)

@ibodor The question about the use cases is something that WP2 is working on. See for example the [use case developed by Team Address](https://ec.europa.eu/cefdigital/wiki/pages/viewpage.action?spaceKey=SDGOO&title=Procedure+10+-+Register+change+of+address) where the criterion “married to a citizen of target MS” is evidenced by a Marriage certificate. So in this case it is indeed about the current marital status.
